core/repository/block.ts

```diff
@@ -216,6 +223,24 @@ class BlockRepo {
 
         return block;
     }
+
+    private async assignTransactions(blocks: Array<Block>): Promise<Array<Block>> {
```

Comment:
I think we need to calculate the sum of transactionCount and if it is equal zero need return blocks